user_guide_src/source/incoming/controllers.rst

```diff
@@ -153,6 +153,9 @@ Auto Routing (Improved)
 
 Since v4.2.0, the new more secure Auto Routing has been introduced.
 
+.. note:: If you are familiar with CodeIgniter 3's Auto Routing or Auto Routing (Legacy),
```

Comment:
From a newcomer perspective - I'm not sure if I would understand that `Auto Routing (Legacy)` is referring to the CI4. This might be a little confusing.

Reply:
Changed the explanation.

Reply:
Thanks.